Helm Charts template testing is failing with `TypeError: attrib() got an unexpected keyword argument 'convert'`

Running `make test` for any Helm Charts for template testing is giving us this [error](https://devops-ci.elastic.co/job/elastic+helm-charts+pull-request+template-testing/186/CHART=elasticsearch,label=docker&&virtual/console):

```

17:55:32 pytest -sv --color=yes
17:55:32 Traceback (most recent call last):
17:55:32   File "/usr/local/bin/pytest", line 10, in <module>
17:55:32     sys.exit(main())
17:55:32   File "/usr/local/lib/python3.7/site-packages/_pytest/config/__init__.py", line 61, in main
17:55:32     config = _prepareconfig(args, plugins)
17:55:32   File "/usr/local/lib/python3.7/site-packages/_pytest/config/__init__.py", line 182, in _prepareconfig
17:55:32     config = get_config()
17:55:32   File "/usr/local/lib/python3.7/site-packages/_pytest/config/__init__.py", line 156, in get_config
17:55:32     pluginmanager.import_plugin(spec)
17:55:32   File "/usr/local/lib/python3.7/site-packages/_pytest/config/__init__.py", line 530, in import_plugin
17:55:32     __import__(importspec)
17:55:32   File "/usr/local/lib/python3.7/site-packages/_pytest/tmpdir.py", line 25, in <module>
17:55:32     class TempPathFactory(object):
17:55:32   File "/usr/local/lib/python3.7/site-packages/_pytest/tmpdir.py", line 35, in TempPathFactory
17:55:32     lambda p: Path(os.path.abspath(six.text_type(p)))
17:55:32 TypeError: attrib() got an unexpected keyword argument 'convert'
17:55:32 make: *** [../helpers/common.mk:17: pytest] Error 1
```

This is a blocker for #307.

--- a/helpers/helm-tester/requirements.txt
+++ b/helpers/helm-tester/requirements.txt
@@ -1,2 +1,10 @@
+atomicwrites==1.3.0
+attrs==19.1.0
+importlib-metadata==0.23
+more-itertools==7.2.0
+pluggy==0.13.0
+py==1.8.0
 pytest==4.1.0
 PyYAML==4.2b4
+six==1.12.0
+zipp==0.6.0

